.github/workflows/ci.yml

```diff
@@ -0,0 +1,206 @@
+  GIT_EMAIL: ${{ secrets.BOT_GITHUB_EMAIL }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+  QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+  TAS_ENVIRONMENT: ./tests/environment
+  TAS_SCRIPTS: ../alfresco-community-repo/packaging/tests/scripts
+  # Release version has to start with real version (7.3.0-....) for the docker image to build successfully.
+  RELEASE_VERSION: 7.2.1
+  DEVELOPMENT_VERSION: 7.2.2-SNAPSHOT
+
+jobs:
+  tas_tests:
+    name: ${{ matrix.testSuite }} TAS tests
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, '[skip tas]') && !contains(github.event.head_commit.message, '[skip tests]') }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - testSuite: REST API part1
+            pom: tests/tas-restapi/pom.xml
+            profiles: all-tas-tests,run-restapi-part1
+            compose-file: docker-compose-minimal+transforms.yml
+          - testSuite: REST API part2
+            pom: tests/tas-restapi/pom.xml
+            profiles: all-tas-tests,run-restapi-part2
+            compose-file: docker-compose-minimal+transforms.yml
+          - testSuite: REST API part3
+            pom: tests/tas-restapi/pom.xml
+            profiles: all-tas-tests,run-restapi-part3
+            compose-file: docker-compose-minimal+transforms.yml
+          - testSuite: CMIS (BROWSER binding)
+            pom: tests/tas-cmis/pom.xml
+            profiles: all-tas-tests,run-cmis-browser
+            compose-file: docker-compose-minimal+transforms.yml
+          - testSuite: CMIS (ATOM binding)
+            pom: tests/tas-cmis/pom.xml
+            profiles: all-tas-tests,run-cmis-atom
+            compose-file: docker-compose-minimal+transforms.yml
+          - testSuite: CMIS (WEBSERVICES binding)
+            pom: tests/tas-cmis/pom.xml
+            profiles: all-tas-tests,run-cmis-webservices
+            compose-file: docker-compose-minimal+transforms.yml
+          - testSuite: Email
+            pom: tests/tas-email/pom.xml
+            profiles: all-tas-tests
+            compose-file: docker-compose-minimal.yml
+          - testSuite: WebDAV
+            pom: tests/tas-webdav/pom.xml
+            profiles: all-tas-tests
+            compose-file: docker-compose-minimal.yml
+          - testSuite: Integration
+            pom: tests/tas-integration/pom.xml
+            profiles: all-tas-tests
+            compose-file: docker-compose-minimal.yml
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.33.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.33.0
```

Comment:
This sets up a Java 17 by default. 7.2.X **need to be compiled with Java 11** Please take a look how it's done here https://github.com/Alfresco/acs-packaging/blob/release/7.2.N/.github/workflows/ci.yml#L18

Reply:
Done